README.md

```diff
@@ -2,54 +2,61 @@
 
 
 ## What do you get out of the box?
+[Read about why we made these technology choices and where they are most applicable.](docs/technology-choices.md)
+
+[Check out some resources for learning more about these technologies.](docs/learning-resources.md)
+
 ### Infrastructure
 - Fully configured infrastructure-as-code AWS environment including:
   - VPCs per environment (staging, production) with pre-configured subnets, security groups, etc.
-  - EKS Kubernetes cluster per environment, pre-configured with tools like cert-manager, external-dns, nginx-ingress-controller
+  - EKS Kubernetes cluster per environment, pre-configured with helpful tools like cert-manager, external-dns, nginx-ingress-controller
   - RDS database for your application (Postgres or MySQL)
   - S3 buckets and Cloudfront distributions to serve your assets
 - Logging and Metrics collected automatically using either Cloudwatch or Prometheus + Grafana, Elasticsearch + Kibana
-- VPN using [Wireguard][wireguard]
+- VPN using [Wireguard][wireguard] (Optional)
 - User management and Identity / Access Proxy using Ory [Kratos][kratos] and [Oathkeeper][oathkeeper] (Optional)
-- Tooling to make it easy to set up secure access for your team
-- Local/Cloud Hybrid developer environment using Skaffold and Telepresence (In Progress)
+- Tooling to make it easy to set up secure access for your dev team
+- Local/Cloud Hybrid developer environment using Telepresence (Optional)
 
 ### Backend
 - Golang or Node.js example project automatically set up, Dockerized, and deployed to your new Kubernetes cluster
-- CI pipeline built with [CircleCI][circleci]. Just merge a PR and a deploy will start. Your code will be built and tested, deployed to staging, then prompt you to push to production
+- CI pipeline built with [CircleCI][circleci] or GitHubActions. Just merge a PR and a deploy will start. Your code will be built and tested, deployed to staging, then prompt you to push to production
 - File upload / download support using signed Cloudfront URLs (Optional)
 - Email support using [SendGrid][sendgrid] or AWS SES (Optional)
 - Notification support for sending and receiving messages in your application (web, mobile, SMS, Email, etc.) (Optional) (In Progress)
 - User management integration with Kratos and Oathkeeper - No need to handle login, signup, authentication yourself (Optional)
-- Feature Flagging integration with [LaunchDarkly][launchdarkly] (Optional) (In Progress)
 
 ### Frontend
 - React example project automatically set up, deployed and served securely to your customers
-- CI pipeline built with CircleCI. Just merge a PR and a deploy will start. Your code will be built and tested, deployed to staging, then prompt you to push to production
+- CI pipeline built with CircleCI or GitHub Actions. Just merge a PR and a deploy will start. Your code will be built and tested, deployed to staging, then prompt you to push to production
```

Comment:
Same comment as before here, I think having a separate category with CI/CD would be better as vendors have made this category popular in the industry.

Reply:
There is a CI/CD section in the tech choices doc, not sure if it's worth getting too deep into it here..